clippy_lints/src/implicit_return.rs

```diff
@@ -1,9 +1,16 @@
-use crate::utils::{in_macro_or_desugar, is_expn_of, snippet_opt, span_lint_and_then};
-use rustc::hir::{intravisit::FnKind, Body, ExprKind, FnDecl, HirId, MatchSource};
-use rustc::lint::{LateContext, LateLintPass, LintArray, LintPass};
-use rustc::{declare_lint_pass, declare_tool_lint};
+use crate::utils::{
+    in_macro_or_desugar, match_def_path,
+    paths::{BEGIN_PANIC, BEGIN_PANIC_FMT},
+    resolve_node, snippet_opt, span_lint_and_then,
+};
+use if_chain::if_chain;
+use rustc::{
+    declare_lint_pass, declare_tool_lint,
+    hir::{intravisit::FnKind, Body, Expr, ExprKind, FnDecl, HirId, MatchSource, StmtKind},
+    lint::{LateContext, LateLintPass, LintArray, LintPass},
+};
 use rustc_errors::Applicability;
-use syntax::source_map::Span;
+use syntax_pos::Span;
```

Comment:
Couldn't you use `syntax::syntax_map::Span` here? Everything else LGTM now.

Reply:
Done. I thought these two were not the same.
Thanks.